website/docs/r/policy_host_transport_node.html.markdown

```diff
@@ -0,0 +1,164 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_host_transport_node"
+description: A resource to configure a HostTransportNode.
+---
+
+# nsxt_policy_host_transport_node
+
+This resource provides a method for the management of a HostTransportNode.
+This resource is supported with NSX 4.1.0 onwards.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_host_transport_node" "test" {
+  description       = "Terraform-deployed host transport node"
+  display_name      = "tf_host_transport_node"
+  site_path         = "/infra/sites/default"
+  enforcement_point = "default"
+
+  node_deployment_info {
+    ip_addresses = ["10.168.186.150"]
+
+    os_type    = "ESXI"
+    os_version = "7.0.3"
+
+    host_credential {
+      username   = "user1"
+      password   = "password1"
+      thumbprint = "thumbprint1"
+    }
+  }
+
+  standard_host_switch {
+    host_switch_mode       = "STANDARD"
+    host_switch_type       = "NVDS"
+    host_switch_profile_id = ["/infra/host-switch-profiles/2812c1e9-00e6-471a-9678-873c101d8f85"]
+
+    ip_assignment {
+      assigned_by_dhcp = true
+    }
+
+    transport_zone_endpoint {
+      transport_zone_id = "/infra/sites/default/enforcement-points/default/transport-zones/1b3a2f36-bfd1-443e-a0f6-4de01abc963e"
+    }
+
+    pnic {
+      device_name = "fp-eth0"
+      uplink_name = "uplink1"
+    }
+  }
+
+  tag {
+    scope = "app"
+    tag   = "web"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `site_path` - (Optional) The path of the site which the Host Transport Node belongs to. `path` field of the existing `nsxt_policy_site` can be used here.
+* `enforcement_point` - (Optional) The ID of enforcement point under given `site_path` to manage the Host Transport Node.
+* `discovered_node_id` - (Optional)  Discovered node id to create Host Transport Node. Specify discovered node id to create Host Transport Node for Discovered Node. This field is required during Host Transport Node create from vCenter server managing the ESXi type HostNode. No need to provide node_deployment_info while creating Host Transport Node for Discovered Node. If both node_deployment_info and discovered_node_id_for_create provided during Host TransportNode create payload then it will create Host Transport Node from the discovered node id provided in this field.
```

Comment:
We can use `ConflictsWith` in schema definition of `discovered_node_id` to make sure it can't be specified together with `node_deployment_info`, in which case documentation doesn't need to care about both being provided

Reply:
Sounds good. Done.